devito/nodes.py

```diff
@@ -6,14 +6,14 @@
 from collections import Iterable, OrderedDict
 
 import cgen as c
-from sympy import Eq
+from sympy import Eq, Indexed, Symbol
```

Comment:
Is there a reason we're not using our internal types here?

Reply:
So this goes back to a question I asked you online -- "Can we assume that we always **only** see our own data types" -- for which there was no certain answer. So I picked the sympy ones, since it's not harmful... I could try using our types (other PR) and see what happens